### DISCUSSION TITLE
I'm using Next.js 13.3 app route, I got self.__next_f.push and .txt file in routed URL

### DISCUSSION THREAD

**Discussion by MangoQuiz** [vercel/next.js#49098] [App Router] [answered]
<img width="1130" alt="스크린샷 2023-05-03 오전 12 55 20" src="https://user-images.githubusercontent.com/131772167/235719939-692a0d52-c443-49fc-9840-4b6464cdf91b.png">
<img width="1122" alt="스크린샷 2023-05-03 오전 12 55 27" src="https://user-images.githubusercontent.com/131772167/235719969-210401da-b3ea-445e-9500-0ba0355a5599.png">
<img width="309" alt="스크린샷 2023-05-03 오전 12 55 37" src="https://user-images.githubusercontent.com/131772167/235720004-74da7054-fcc2-41d0-8ebb-daef74aadcdc.png">
<img width="376" alt="스크린샷 2023-05-03 오전 12 59 30" src="https://user-images.githubusercontent.com/131772167/235720749-ea34f968-3c86-47e4-b55e-21656d15c74e.png">

Hi there. I'm using Next.js version 13.3. 

I built with output: "export" for SSG, and I keep getting txt after the URL whenever I app route.
(npm run build -> npm run start) 

This is my a tag in first HTML page.
<img width="672" alt="스크린샷 2023-05-03 오전 1 00 02" src="https://user-images.githubusercontent.com/131772167/235720937-bed9997b-2ed1-4731-b4ad-b92e00a0301f.png">

Is there any way to fix this?

**Comment by dhruvilxcode** [comment]
I am getting the same issue.

**Comment by timneutkens** [comment]
Replied here: https://github.com/vercel/next.js/issues/48996#issuecomment-1532693355

**Comment by leerob** [comment, marked as answer]
https://github.com/vercel/next.js/discussions/42170#discussioncomment-8137079
